src/mini/ready.html

```diff
@@ -99,6 +99,11 @@
     },
 
     _ready: function() {
+      // install host attributes
+      this._marshalHostAttributes();

```

Comment:
Suggest we move this into `_configure` since this is the common spot for marshaling element inputs.